src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+
+@Entity
+@Table(name = "users",
+        uniqueConstraints = @UniqueConstraint(columnNames = "email"))
```

Comment:
you also can add a property  `unique = true` to the column email  instead of this property 
```suggestion
        uniqueConstraints = @UniqueConstraint(columnNames = "email"))
```